iron-autogrow-textarea.html

```diff
@@ -69,7 +65,7 @@
 
     <!-- size the input/textarea with a div, because the textarea has intrinsic size in ff -->
     <div class="textarea-container fit">
-      <content id="textarea" select="textarea"></content>
+      <textarea id="textarea" required$="[[required]]" maxlength$="[[maxlength]]"></textarea>
```

Comment:
Can you add a `rows$="[[rows]]"` binding, so that the "initial number of rows" example works?